lib/pacio_test_kit/interactions_test.rb

```diff
@@ -9,7 +30,7 @@ def read_and_validate_resources(resource_ids, tag = '')
         fhir_read(resource_type, id, tags: [tag])
 
         status = request.response[:status]
-        next unless validate_status(status, req_num)
+        next unless validate_status('read', status, req_num)
```

Comment:
Since we don't use this validate_status method for the create, do we want to make this change or keep as original? Currently just removed it for now (kept as original)